docs/protos/proto-sidebars.js

```diff
@@ -1,3 +1,24 @@
-
-module.exports = {"protodocs":[{"type":"category","label":"Files","items":[{"type":"category","label":"/mgmt/v1alpha1","items":[{"type":"doc","id":"mgmt/v1alpha1/api_key.proto"},{"type":"doc","id":"mgmt/v1alpha1/auth.proto"},{"type":"doc","id":"mgmt/v1alpha1/connection.proto"},{"type":"doc","id":"mgmt/v1alpha1/connection_data.proto"},{"type":"doc","id":"mgmt/v1alpha1/transformer.proto"},{"type":"doc","id":"mgmt/v1alpha1/job.proto"},{"type":"doc","id":"mgmt/v1alpha1/metrics.proto"},{"type":"doc","id":"mgmt/v1alpha1/user_account.proto"}]}]}]};
-  
\ No newline at end of file
+module.exports = {
```

Comment:
Did this file get formatted somehow? It should be autogenerated to this format and probably shouldnt have changed.